client/my-sites/plugins/plugin-details-header/index.jsx

```diff
@@ -42,9 +42,10 @@ const PluginDetailsHeader = ( {
 	} );
 
 	// Rating can be a valid number, 0 or null, discard undefined for easier comparison
-	const rating = isMarketplaceProduct
-		? ( reviewsStats?.ratings_average * 100 ) / 5 ?? 0
```

Comment:
This, `reviewsStats?.ratings_average` will return `NaN` for marketplace ratings as `reviewsStats` hasn't been resolved yet. This results in a brief flash of `null` in the view.